dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1741,6 +1741,12 @@
 		</ResolveNativeReferences>
 	</Target>
 
+	<Target Name="_ValidateOutputType"
+		Condition="'$(_PlatformName)' == 'macOS' Or '$(_PlatformName)' == 'MacCatalyst' "
```

Comment:
WinExe is never valid for our projects, so the condition is unnecessary.

However, you need a condition on the `OutputType`, so you only show this error when OutputType=WinExe.